mavenlint.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |s|
   s.name        = 'mavenlint'
-  s.version     = '1.6.0'
+  s.version     = '1.7.0'
```

Comment:
I'm bumping a minor version for the dependencies changes we can leave 1.6 to be a way to adjust projects before Ruby 3 and 1.7 and beyond is aimed to fix Ruby 3+ projects.